Assignment2/PabloB94-Pablo Beltran de Casso.rdf

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<rdf:RDF xml:base="http://www.ontogrid.net/StickyNote#"
+    xmlns="http://www.ontogrid.net/StickyNote#"
+    xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#">
+	
+ <rdf:Description rdf:about="#Class01">
+ 	<includes rdf:resource="#Sensor029"/>
+ 	<includes rdf:resource="#Computer101"/>
+ </rdf:Description>
+ 
+ <rdf:Description rdf:about="#Sensor029">
+ 	<hasMeasurement rdf:resource="#Measurement8401"/>
+ </rdf:Description>
+ 
+ <rdf:Description rdf:about="#Measurement8401">
+ 	<hasTemperature rdf:datatype="http://www.w3.org/2001/XMLSchema#temperature">29</hasTemperature>
+ 	<atTime rdf:datatype="http://www.w3.org/2001/XMLSchema#time">2010-06-12T12:00:12</atTime>
+ </rdf:Description>
+ 
+ <rdf:Description rdf:about="#Computer101">
+ 	<hasOwner rdf:resource="#User10A"/>
+ </rdf:Description>
+ 
+ <rdf:Description rdf:about="#User10A">
+ 	<hasName rdf:datatype="http://www.w3.org/2001/XMLSchema#name">Pedro   </hasName>
```

Comment:
Be careful with spaces. I have already commented to one of the other students, since the problem is the same because of copying and pasting.